src/examples/java/MessageListenerExample.java

```diff
@@ -87,13 +86,13 @@ public void onMessageReceived(MessageReceivedEvent event)
         User author = event.getAuthor();                //The user that sent the message
         Message message = event.getMessage();           //The message that was received.
         MessageChannel channel = event.getChannel();    //This is the MessageChannel that the message was sent to.
-                                                        //  This could be a TextChannel, PrivateChannel, or Group!
+                                                        //  This could be a channel from a guild (TextChannel or a StoreChannel) or a PrivateChannel!
```

Comment:
Last I checked store channels are not message channels.

Reply:
Yeah, my bad, I missread the docs. I saw both `StoreChannel` and `TextChannel` extend `GuildChannel` and confused that with `MessageChannel`. Guess I also confused the `StoreChannel` with the News / Announcements Channel. Sorry for that, I'll revert that.